lib/ui/channel_buffers.dart

```diff
@@ -0,0 +1,83 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of dart.ui;
+
+/// A saved platform message for a channel with its callback.
```

Comment:
This needs to be fleshed out a bit more. Do I ever want to use this as a user? If so, why and when?

I'm also not sure `StoredMessage` is the right name - maybe `StoredPlatformChannelMessage`?

Reply:
It is just an immutable tuple in 5 lines of code.  It doesn't have any logic, no public methods take it as a parameter, it's just a datatype.  IMO It should be clear if someone wanted to use it.  I would have used a tuple class if it was available with standard dart.